src/lib/model/OutboundTransfersModel.js

```diff
@@ -714,7 +716,7 @@ class OutboundTransfersModel {
     async load(transferId) {
         try {
             const data = await this._cache.get(`transferModel_${transferId}`);
-            if(!data) {
+            if (!data) {
```

Comment:
please dont change whitespace unless there is a very good reason to do so.

Reply:
Deleted